src/agent/Core/SpawningKit/Spawner.h

```diff
@@ -871,8 +872,43 @@ class Spawner {
 		}
 	}
 
+	void enterLveJail(const struct passwd * pw)
```

Comment:
Please put brackets right after the function name, not on a new line, as per the [Passenger C++ coding style](https://github.com/phusion/passenger/blob/stable-5.0/CONTRIBUTING.md#cxx_coding_style):

    void enterLveJail(const struct passwd * pw) {